src/chunkserver/clone_copyer.cpp

```diff
@@ -173,9 +217,20 @@ void OriginCopyer::DownloadFromCurve(const string& fileName,
     int fd = 0;
     {
         std::unique_lock<std::mutex> lock(mtx_);
+        std::unique_lock<std::mutex> lockTime(timeMtx_);
         auto iter = fdMap_.find(fileName);
         if (iter != fdMap_.end()) {
             fd = iter->second;
+            auto fdIter = std::find_if(
```

Comment:
since the curveOpenTime_ list is orderd, maybe we can use binary search to accelerate.